liquidctl/driver/gigabyte_rgb_fusion.py

```diff
@@ -0,0 +1,328 @@
+"""liquidctl drivers for Gigabyte RGB Fusion via ITE Tech USB Device 0x5072
+
+RGB Fusion 2.0
+--------------
+Gigabyte RGB Fusion 2.0 is a lighting controller that supports 12V RGB and
+5V Addressable RGB lighting effects. It is built into motherboards that contain
+the RGB Fusion 2.0 logo, and supports both (a) RGB/ARGB lighting elements on
+the motherboard itself, (b) RGB/ARGB elements on memory modules, and 
+(c) RGB/ARGB elements connected to 12V RGB and 5V ARGB headers.
+
+Lighting is controlled on these motherboards via an ITE Tech chip with 
+product ID 0x5072 that is mapped to a USB 2 port. On the Gigabyte Z490 Vision D,
+for example, the lighting controller is mapped to USB port HS12.
+
+Driver
+------
+
+This driver implements the following features available at the hardware level:
+
+ - initialization
+ - control of lighting modes and colors
+ - reporting of firmware version
+
+The driver supports 7 color channels and a 'sync' channel. Channel names must
+be specified exactly as shown (upper/lower case matters):
+ - IOLED    : This is the LED next to the IO panel
+ - LED1     : This is one of two 12V RGB headers
+ - PCHLED   : This is the LED on the PCH chip ("Designare" on Vision D)
+ - PCILED   : This is an array of LEDs behind the PCI slots on *back side* of motherboard
+ - LED2     : This is second 12V RGB header
+ - DLED1    : This is one of two 5V addressable RGB headers
+ - DLED2    : This is second 5V addressable RGB header
+
+Each of these channels can be controlled individually. However, channel name 'sync'
+can be used to control all 7 channels at once.
+
+The driver supports 6 color modes:
+ - Off
+ - static
+ - pulse
+ - flash
+ - double-flash
+ - color-cycle
+ 
+The more elaborate Addressable RGB color/animation schemes permissable on DLED1
+and DLED2 headers are not currently supported.
+
+For color modes pulse, flash, double-flash and color-cycle, the speed of color change
+is governed by the --speed parameter on command line. It may be set to:
+ - slowest
+ - slower
+ - normal (default)
+ - faster
+ - fastest
+ - ludicrous
+
+The driver also supports brightness levels via --brightness argument on the command
+line. Brightness is specified in percentage from 0 to 100. Example:
```

Comment:
For the most part I would rather _not_ have a separate "brightness" dial because it adding duplication.  The CLI [already supports HSV and HSL color formats](https://github.com/jonasmalacofilho/liquidctl#supported-color-specification-formats), and if we were to add a forth RGB+brightness format it would be much more interesting to do so there, generically and usable by all drivers.

On the other hand, this is supported at the firmware level, so I'm not sure.  Maybe leave it at the API level but don't expose it on the CLI?

Reply:
The target audience of this driver will not be familiar with HSV or HSL. For them, even constructing colors as GGBBRR can be challenging. A "brightness" argument improves ease-of-use and is directly supported by the firmware.

If we don't expose it on the CLI, how will users specify it?

Reply:
You might be right that the audience (it's the same whether we're talking about this driver or other liquidctl drivers) might not necessarily be familiar with HSV or HSL, but those are the better solution: they allows you to parametricaly vary hue, saturation _or_ value/luminosity, instead of just one of them; also, each of them can be varied by individual color, instead of globally for the entire command.  So maybe we should improve the documentation of HSL/HSV?

Another big problem is that a new `--brightness` argument that only works with a single driver is bound to cause me a _lot_ of headache.  There's no imaginable reason (especially for this hypothetical uses that isn't familiar with HSV/HSL) why it would only work with a single driver.  Just imagine all issues titled "--brightness does nothing"...

> If we don't expose it on the CLI, how will users specify it?

By that I really meant _not_ exposing it to users, only keeping it in the API for some future use (and for completeness of the driver).

TL;DR: I don't think `--brightness` is a good interface for the users.  Anything that looks to be that general has to be supported by in all drivers.  I also think it should be a forth color format mode instead of an argument, because it's useful to vary brightness per color instead of only per channel/command.

Reply:
Agreed. Exposing it on the CLI will indeed result in a lot of "brightness does nothing" complaints.

Also, it's easy to reduce brightness just by lowering the value of each color component in GGBBRR.

I've removed `brightness`.